src/components/TodoFooter.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <footer class="footerEnd">
```

Comment:
클래스명은 케밥 케이스를 사용해야합니다.
https://velog.io/@jp-share/Coding-%ED%91%9C%EA%B8%B0%EB%B2%95-%EC%B9%B4%EB%A9%9C%EC%BC%80%EC%9D%B4%EC%8A%A4-%ED%8C%8C%EC%8A%A4%EC%B9%BC%EC%BC%80%EC%9D%B4%EC%8A%A4-%EC%8A%A4%EB%84%A4%EC%9D%B4%ED%81%AC%EC%BC%80%EC%9D%B4%EC%8A%A4
```suggestion
    <footer class="footer-end">
```